apps/calendar/js/views/month.js

```diff
@@ -105,12 +108,25 @@ Month.prototype = {
         this.controller.selectedDay = date;
         break;
       case 'dbltap':
-        this.app.go('/day/');
+        // make sure we discard double taps that started on a different day
+        if (this._lastTarget === target) {
+          this._goToAddEvent();
```

Comment:
also doesn't change anything since target is only checked inside `dbltap` and `click` always dispatches on the first tap (which will override the `_lastTarget`)